openmicroscopy/openmicroscopy
# Thumbnail cache busting

We now load the latest thumbnail version for web thumbnails and use this in the thumb src, so that if a new thumbnail is created, the browser-cached thumbnail will be replaced.

See https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=7593

To test:
- Log in to webclient and view dataset of thumbnails
- Log in to Insight, edit rendering settings for one of the images in the dataset. Save.
- Refresh the dataset in web (refresh button above tree) - thumbnail should update.

This includes new and updated methods in the Blitz Gateway and tests for these.

--no-rebase
